src/Http/routes.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-Route::group(['prefix' => 'api/exponent/devices', 'middleware' => ['auth:api', 'bindings']], function () {
+Route::group(['prefix' => 'api/exponent/devices', "middleware" => 'expo.middleware'], function () {
```

Comment:
Can you change the double quotes to single?

Reply:
Done.